Bootstrap 5.3.5

agriffard opened the pull request:
https://github.com/twbs/bootstrap/releases/tag/v5.3.5
https://www.npmjs.com/package/bootstrap/v/5.3.5

gvkries commented:
I also had these unexpected differences. You need to remove the outdated `node_modules` folders from the root of the affected projects.

agriffard commented:
Deleted my `node_modules` folder.

Updated to Bootstrap v5.3.5 but `yarn build -gr` returns errors so I can't build all the assets.

For example, on the asset `yarn build -n theme-theadmin` : 

`[theme-theadmin] 1 │ @import "../../../../../../node_modules/bootstrap-select/sass/bootstrap-select";`

`bootstrap-select` is indeed not restored by `yarn` command.

Can someone please try?

agriffard commented:
> I also had these unexpected differences. You need to remove the outdated `node_modules` folders from the root of the affected projects.

Deleted my `node_modules` folder.

Updated to Bootstrap v5.3.5 but the check is still failing.

gvkries commented:
Hmm, that has worked for me. Did you delete the nodes folder e.g. in `src/OrchardCore.Modules/OrchardCore.Apis.GraphQL`? I don't mean the main `nodes_modules` folder in the solution root.

agriffard commented:
> Hmm, that has worked for me. Did you delete the nodes folder e.g. in `src/OrchardCore.Modules/OrchardCore.Apis.GraphQL`? I don't mean the main `nodes_modules` folder in the solution root.

Thank you, deleted all `node_modules` folders and updated yarn to 4.8.1 and no more errors after build.
